src/components/VmDetails/cards/UtilizationCard/NetworkingCharts.js

```diff
@@ -82,7 +90,7 @@ const NetworkingCharts = ({ stats, isRunning }) => {
   )
 }
 NetworkingCharts.propTypes = {
-  stats: PropTypes.object.isRequired,
+  netStats: PropTypes.object.isRequired,
```

Comment:
If there is can be a case when netStats is `null` or `undefined`, then `isRequired` need to be removed.

Reply:
There cannot be that case.  The chart components require at least the current set of stat data.  They can handle the history not being there, but they cannot handle no stats.  The base card handles if the stats have not been loaded or are missing after a load attempt.

(See **UtilizationCard/index.js**)

Reply:
Then why do you check `netStats` on line 31?
`const haveNetworkStats = netStats && netStats['current.total']`

Reply:
Because I missed cleaning it up in the last commit.

The Networking statistics isn't always available/present in the api call, but there is always a placeholder for it in the internal `VmStatistics` object.  So the object the index component checks is good but the network component needs to see if the stats it needs are present.

Not doing this check for CPU and Memory since those stats have been present for a long time.  Network stats is new in 4.3.

Reply:
So if I understand you right this part `netStats &&` is not needed.